pkg/lib/config/configsource/resources.go

```diff
@@ -538,6 +538,10 @@ func (d AuthgearSecretYAMLDescriptor) UpdateResource(ctx context.Context, _ []re
 		// Since the key neither use for sig nor enc
 		GenerateClientSecretOctetKeyFunc: secrets.GenerateOctetKey,
 		GenerateAdminAPIAuthKeyFunc:      secrets.GenerateRSAKey,
+		GenerateSAMLIdpSigningCertificate: func() (*config.SAMLIdpSigningCertificate, error) {
+			// FIXME: The common name is not important, but we might want to put the app id here.
```

Comment:
What value does other competitor put in here?

Reply:
It is something like an id.

Reply:
I suggest using `APP_HOST_SUFFIX` (which is something like `.authgear.cloud`) to derive the common name here.

Reply:
Updated to use the saml entity id here.